Quran/AyahInfoPersistenceStorage.swift

```diff
@@ -0,0 +1,38 @@
+//
+//  AyahInfoPersistenceStorage.swift
+//  Quran
+//
+//  Created by Ahmed El-Helw on 5/12/16.
+//  Copyright © 2016 Quran.com. All rights reserved.
+//
+
+import Foundation
+import SQLite
+
+struct AyahInfoPersistenceStorage {
+
+    private let db: Connection
```

Comment:
In general, I'm against singleton. So, I suppose the latter is the way to go since we will only need one that will be owned indirectly by a view controller.